static/upnp/upnp-on-http-api2.0.xml

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0"?>
+<scpd xmlns="urn:schemas-upnp-org:service-1-0" configId="1">
+    <specVersion>
+        <major>1</major>

```

Comment:
Keep in mind that UPnP expects SOAP based API definitions;  these service descriptors are merely setup to satisfy 1.1 compatibility.